packages/cozy-harvest-lib/src/policies/biWebView.spec.js

```diff
@@ -269,43 +262,43 @@ describe('fetchExtraOAuthUrlParams', () => {
     expect(result).toMatchObject({ token: 'bi-temporary-access-token-123' })
   })
 
-  it('should add id_connector param for multiple bank ids', async () => {
+  it('should add connector_uuids param for multiple bank uuids', async () => {
     const client = new CozyClient({
       uri: 'http://testcozy.mycozy.cloud'
     })
     client.query = jest.fn().mockResolvedValue({
       data: {
         timestamp: Date.now(),
-        code: 'bi-temporary-access-token-123',
-        biMapping: { [TEST_BANK_COZY_ID]: 2 }
```

Comment:
Does that mean that we stored this `biMapping` inside the account? Should we remove it now?

Reply:
This will be done by the banking konnector : https://gitlab.cozycloud.cc/scrapers/banking-konnectors/-/merge_requests/431